src/components/Post/GovernanceSideBar/index.tsx

```diff
@@ -698,8 +700,13 @@ const GovernanceSideBar: FC<IGovernanceSidebarProps> = (props) => {
 									</GovSidebarCard>
 									:
 									<GovSidebarCard className='flex items-center'>
-										You Voted: { lastVote == 'aye' ? <LikeFilled className='text-aye_green ml-2' /> : <DislikeFilled className='text-nay_red ml-2' /> }
+
+										You Voted: { lastVote == 'aye' ? <LikeFilled className='text-aye_green ml-2' /> : ''}
```

Comment:
for now just showing last vote with the icon , UI will be Updated

Reply:
Is this updated ?

Reply:
Not yet but it would be updated in a while, so for now just showing this , discussed it with param